app/actions.py

```diff
@@ -38,7 +38,12 @@ def __init__(self):
     app.mutator.Mutator.__init__(self)
     self.view = None
     self.rootGrammar = app.prefs.getGrammar(None)
+    # |__skipUpdateScroll| is False if you do not want the program
+    # to automatically set the view to the optimal scroll position.
     self.__skipUpdateScroll = False
```

Comment:
If there's an opportunity to remove these __skip* flags, please go for it. I'd prefer not to have them, but I understand it can be hard to avoid at times.

Reply:
I actually think we do need them just because our updateScrollPosition function is called upon refreshing the screen, and sometimes we don't want to update the scroll position (specifically for if captive cursor is false). I refactored some functions, so now the __useOptimalScroll flag tells the function if it should use the basic or optimal scroll position algorithm. I don't see a real easy way to get rid of these flags for now.